CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ to review any possible breaking changes to direct/custom use of ag-Grid APIs and
 ### 🎁 New Features
 
 * `AppOption` configs now accept an `omit` property for conditionally excluding options.
+* Grid column `flex` prop will now accept numbers, with available space divided between flex columns
```

Comment:
not to be a stickler but this is a 'property', or 'config' or 'param' anything really other than 'prop'  :-)

Reply:
Right you are.